routes/web.php

```diff
@@ -43,6 +43,11 @@
 });
 
 Route::prefix('user')->group(function () {
+    Route::get('check/', 'UserController@check')->name('quiz.check');
```

Comment:
this technically works, but is poorly designed. We are trying to follow RESTful design which basically we implement the HTTP verbs.

```
POST - create
GET - read
PUT - update/replace
PATCH - update/modify
DELET - deletes
```

inside your check function you are trying to store data which irrelevant to the route method `get` it should  be read only and separate a function the will handle the storing part